src/daft-csv/src/read.rs

```diff
@@ -136,6 +138,49 @@ pub fn read_csv_bulk(
     tables.into_iter().collect::<DaftResult<Vec<_>>>()
 }
 
+#[inline]
+fn assert_stream_send<'u, R>(
+    s: impl 'u + Send + Stream<Item = R>,
+) -> impl 'u + Send + Stream<Item = R> {
+    s
+}
+
+// Parallel version of table concat
+// get rid of this once Table APIs are parallel
+fn parallel_table_concat(tables: &[Table]) -> DaftResult<Table> {
+    if tables.is_empty() {
+        return Err(DaftError::ValueError(
+            "Need at least 1 Table to perform concat".to_string(),
+        ));
+    }
+    if tables.len() == 1 {
+        return Ok((*tables.first().unwrap()).clone());
```

Comment:
We could elide this clone by taking an owned `Vec`, which the caller should be able to supply, and returning `Ok(tables.into_iter().next().unwrap())` here, which should allow us to move the single table element into the return value.